src/app/learn/get-started/basics/build-publish/index.html

```diff
@@ -0,0 +1,43 @@
+<stache
+  navOrder="7"
+  pageTitle="Build and Publish a SPA"
+  navTitle="Build and publish">
+
+  <stache-page-summary>
+    <p>
+      After you update the SKY UX CLI with the latest version, you can build and publish your SPA. 
+    </p>
+    <sky-alert alertType="info">
+      Blackbaud developers should create repos and provision applications through the SKY UX App Management Portal. Documentation on the SKY UX App Management Portal is available on <a href="https://host.nxt.blackbaud.com/engineering-system-docs/learn/spa/spa-app-portal">the Engineering System website</a>. This site is for Blackbaud internal use only and is not available to external developers.
+    </sky-alert>
+  </stache-page-summary>
+  
+  <ol>
+    <li>
+      <p>
+        From the command prompt, <stache-code>cd</stache-code> into the application's directory.
+      </p>
+    </li>
+    <li>
+      <p>
+        Enter <stache-code>skyux build</stache-code> to build the SPA into the <stache-code>dist/</stache-code> folder.
+      </p>
+    </li>
+    <li>
+      <p>
+        Copy the contents of the <stache-code>dist/</stache-code> folder into a directory such as <stache-code>my-spa/</stache-code>, where<stache-code>my-spa</stache-code> is the name of your SPA.
```

Comment:
Can we say something like, "Based on the name you provided when running `skyux new` (and cross-link to that step), rename the `dist` folder to that name.  For example, if you provided "my-spa", you would rename the `dist/` folder to `my-spa/`.